lib/guest_installation_and_configuration_base.pm

```diff
@@ -1591,6 +1624,20 @@ sub config_guest_installation_automation_registration {
     return $self;
 }
 
+# Configure guest sysinfo
+sub config_guest_sysinfo {
+    my $self = shift;
+
+    $self->reveal_myself;
+    $self->config_guest_params(@_) if (scalar(@_) gt 0);
+
+    if ($self->{guest_sysinfo} ne '') {
+        $self->{guest_sysinfo_options} .= " --sysinfo $self->{guest_sysinfo} ";
+        record_info("Guest $self->{guest_name} has been set sysinfo.");
```

Comment:
Why not make record_info more informative as below ?
```
record_info("Guest $self->{guest_name} has been set sysinfo", "The sysinfo is $self->{guest_sysinfo}");
```

Reply:
This will show in the dry run. Duplicated info can be avoided, imho.